week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,31 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp() {
+    this.volume += 100; // calling this function should pour your bottle full (volume = 100);
   },
-  pour: function () {
-    // calling this function should increase your bottle volume by 10 unit;
+  pour() {
+    if (this.volume <= 90) {
+      return (this.volume += 10); // calling this function should increase your bottle volume by 10 unit;
+    }
   },
-  drink: function() {
-    // calling this function should decrease your bottle volume by 10 unit;
+  drink() {
+    if (this.volume >= 10) {
+      return (this.volume -= 10); // calling this function should decrease your bottle volume by 10 unit;
+    }
   },
-  isFull: function () {
-    // this function should return true if your bottle is empty;
-  },
-  isEmpty: function() {
+  isFull() {
     // this function should return true if your bottle is full;
-  }
+    if (this.volume >= 100) {

```

Comment:
First, can we have more than a 100 ml?
Second? do you need the condition if you're checking a boolean
so what about returning 
```return this.volum===100```

Reply:
Understood, changed as recommended. Changed isEmpty functions as well.
I will look how to use boolean again, since I keep trying to check those with if condition, unnecessarily.